libs/core/langchain_core/output_parsers/list.py

```diff
@@ -9,24 +9,26 @@
 from typing import Optional as Optional
 from typing import TypeVar, Union
 
+from typing_extensions import override
+
 from langchain_core.messages import BaseMessage
 from langchain_core.output_parsers.transform import BaseTransformOutputParser
 
 T = TypeVar("T")
 
 
-def droplastn(iter: Iterator[T], n: int) -> Iterator[T]:
+def _droplastn(iterator: Iterator[T], n: int) -> Iterator[T]:
```

Comment:
Is it OK to consider `droplastn` private ?

Reply:
I generally suggest avoiding doing so unless there's a good reason -- we didn't have an official policy about the public interface, so it's hard to know what user code might be relying on, and unclear that there's a good reason to risk breaking some users code

Reply:
OK. I understand.
As a user though, I would really not use a function from `output_parsers.list` in my own code...

Reply:
I rolled back. PTAL.

Reply:
Hi @eyurtsev could you take another look ?